File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        int j = 1;
        
        while(j < nums.length){
            if(nums[i] == nums[j]){
                j++;
            }else{
                nums[i + 1] = nums[j];
                i++;
            }
        }
        return i + 1;
    }
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let i =0;j=1;
    while(j < nums.length){
        if(nums[i] === nums[j]){
            j++;
        }else{
            nums[i+1] = nums[j];
            i++
        }
    }
    return i+1;
};